### PR TITLE
feat(bitbucket-server): fetch only own pr's

### PR DESCRIPTION
**Pull request by viceice**
This pr add's a user filter to the `getPrList` request to only fetch the renovate pr's.

Not yet live tested. The earliest i can do it is tomorrow.

### CODE REVIEW

**Comment by iamstarkov** [comment]
what is the objective or reasoning?

File: lib/platform/bitbucket-server/index.ts [hunk 1]
```diff
@@ -623,8 +626,13 @@ export async function getPrList(_args?: any) {
   logger.debug(`getPrList()`);
   // istanbul ignore next
   if (!config.prList) {
+    const query = new URLSearchParams({
+      state: 'ALL',
+      'role.1': 'AUTHOR',
+      'username.1': config.username,
```
Comment (iamstarkov) [inline lib/platform/bitbucket-server/index.ts]: are `role.1` and `username.1` legit query params?

Comment (ViceIce) [reply lib/platform/bitbucket-server/index.ts]: Yes, the web ui is using the same query. [docu](https://docs.atlassian.com/bitbucket-server/rest/5.0.0/bitbucket-rest.html#idm45659054385280)

You can add multiple, up to ten filters.

**Comment by ViceIce** [comment]
Tested live now, is working fine 🎉

File: lib/platform/bitbucket-server/index.ts (hunk 1 above)
Comment (iamstarkov) [reply lib/platform/bitbucket-server/index.ts]: good

**Review by iamstarkov — APPROVED** [review]
I guess it is a perf optimization. if it is, then its cool

**Comment by ViceIce** [comment]
yes, just a perf optimization. we only want to load our own pullrequests.

**Pull request merged**